code/__HELPERS/game.dm

```diff
@@ -223,7 +223,7 @@
 					. |= M		// Since we're already looping through mobs, why bother using |= ? This only slows things down.
 	return .
 
-#define SIGN(X) ((X<0)?-1:1)
+#define SIGN_CM(X) ((X<0)?-1:1)
```

Comment:
The difference I suppose is this one will say SIGN(0) is 1. Don't have both of these but handle that wherever that functionality was depended on.

Reply:
I removed SIGN_CM and typed its math on the three only istances of game.dm it was called in, for the time being. Also removed a left over #undef SIGN, cause of many undefined procs errors.